apps/api/v1/pages/api/event-types/_post.ts

```diff
@@ -275,11 +275,17 @@ async function postHandler(req: NextApiRequest) {
     children: _,
     ...parsedBody
   } = schemaEventTypeCreateBodyParams.parse(body || {});
-
+  if (parsedBody.teamId && parsedBody.parentId) {
+    //parentId and teamId can't be used together , parentId is used for child event types which belong to user
+    //teamId is used for event types belonging to a team
+    //set teamId to null to avoid unexpected logical error
+    req.body.teamId = null;
+    parsedBody.teamId = null;
```

Comment:
Why not throw an error with that reason instead? I don't think we should mess with requests directly, let the users handle it.

Reply:
Done!